profile/profile.md

```diff
@@ -0,0 +1,71 @@
+
+## Skills
+
+**Proficient:** Java/Lambda & Stream API, Linux/Bash, SQL, Matlab, Wireshark & Simulink, Agile/Scrum, Git
+
+**Competent:** RDBMS, Springboot, Mockito/Junit, REST APIs, Python, C++/C, Computer Networking
+
+**Familiar:** Google Cloud Platform, Visual Studios, VHDL, ARM Assembly, Machine Learning, Spark & Hadoop, HTML
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_demo](https://github.com/jarviscanada/jarvis_data_eng_demo)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_RoshanRamesh/tree/master/linux_sql)]: Developed a real time cluster monitoring bash agent that collects and records hardware specifications and node usage information(CPU/memory). PostgreSQL database was used to store information of all nodes, provisioned by docker containers. Cronjob was used for automatic resource usage collection and SQL queries were written to account for business questions.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_RoshanRamesh/tree/master/core_java)]:
+      
+  - Twitter App: Implemented a Java8 based TwitterCLI application that interacts via REST API using HTTP requests. The application helps users perform CRUD operations on Twitter posts. Implementation made possible by Springboot along with Junit/Mockito testing libraries.
+  - JDBC App: Developed JDBC based application which bridges RDBMS and the Java application. Users can connect to PSQL database to perform CRUD operations using the DAO pattern.
+  - Grep App: Designed Java 8 Stream APIs with lambda functions that uses pattern matching to search recursively through files in a directory and outputs the matched lines into a new file.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_RoshanRamesh/tree/master/springboot)]: Developed a three-tier Springboot application based on REST API, using Java 8 which simulates an online stock trading platform. The trading platform used IEX cloud as datasource and PSQL database to persist data. Spring DAO was used to handle data access in PSQL. Junit and Mockito were used for testing/integration along with docker to dockerize the final application.
```

Comment:
Developed a three-tier REST API application using Springboot (not Java 8 == core java)
talk more about how the application works, such as data flow